.github/CODEOWNERS

```diff
@@ -1,4 +1,5 @@
-* @cyberark/conjur-core-team @conjurinc/conjur-core-team @conjurdemos/conjur-core-team
+* @cyberark/community-and-integrations-team
+* @cyberark/palmtree
```

Comment:
Format I think requires them to be on the same line: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file

Reply:
fixed - which means this would be good to have a squash merge on

Reply:
can we trim the other lines to have just 1 repetition of the reviewer team?